spec/integration/benchmarks_spec.rb

```diff
@@ -19,15 +19,17 @@
         let(:'X-RH-IDENTITY') { encoded_header }
         schema type: :object,
                properties: {
-                 meta: { '$ref' => '#/definitions/metadata' },
-                 links: { '$ref' => '#/definitions/links' },
```

Comment:
`definitions` are not allowed anymore. https://swagger.io/specification/#components-object